.drone.yml

```diff
@@ -593,9 +641,9 @@ depends_on:
 
 trigger:
   ref:
-  - refs/heads/master
-  - "refs/tags/**"
-  - "refs/pull/**"
+    - refs/heads/master
```

Comment:
Why change this?

Reply:
@lunny It’s auto format by the editor.

Reply:
So your formatter is different from me. :(

Reply:
@lunny Actually, I did nothing with the config, as it's the default in Goland 😄If it's not ok to change, I will revet.